modules/admin_manual/pages/maintenance/upgrading/upgrade_php.adoc

```diff
@@ -7,14 +7,8 @@
 == Introduction
 
 You should, almost, always upgrade to the latest version of PHP supported by ownCloud, if and where possible. 
-And if you're on a version of PHP older than 5.6 you *need* to upgrade. 
-This guide steps you through upgrading your installation of PHP to version 7.0, 7.1, 7.2 or 7.3 on Red Hat or CentOS 7.
-
-:from-version: 5.6
-:to-version: 7.0
-:to-pkg-version: 70
-
-include::{partialsdir}/maintenance/upgrading/upgrade_steps.adoc[leveloffset=+1]
+And if you're on a version of PHP older than 7.1 you *need* to upgrade.
```

Comment:
Is the `And` necessary?
Just writing `If you are ...`
Btw, is it `need` or `must` ? 7.0 was dropped in 10.4 - or?